lib/rucio/db/sqla/migrate_repo/versions/f85a2962b021_adding_transfertool_column_to_requests_.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2013-2020 CERN for the benefit of the ATLAS collaboration.
```

Comment:
Merge the two alembic files

Reply:
unfortunately this is not resolved

Reply:
Ah yes.  I misunderstood this as merging your original alembic files with mine.  Yes we can combine the column and index creation files.